### PR TITLE
261 set boundary circle latlong to point latlon

### LINKED ISSUE #261: `boundary.circle.lat` `boundary.circle.lon` for `/reverse` should be equal to `point.lat` and `point.lon`
`boundary.circle.lat` `boundary.circle.lon` for `/reverse` should be equal to `point.lat` and `point.lon` Also always set `boundary.circle.radius` to the default if not provided.

### PR DESCRIPTION
**Pull request by stephenkhess**
Fixed #261

### CODE REVIEW

File: sanitiser/_geo_common.js [hunk 1]
```diff
@@ -55,6 +56,7 @@ function sanitize_rect( key_prefix, clean, raw, bbox_is_required ) {
  * @param {bool} circle_is_required
  */
 function sanitize_circle( key_prefix, clean, raw, circle_is_required ) {
+  // "boundary.circle", clean, raw, false
```
Comment (dianashk) [inline sanitiser/_geo_common.js]: the info in this comment should be moved to the doc block above the function definition.

File: sanitiser/_geo_reverse.js [hunk 2]
```diff
@@ -10,19 +11,31 @@ module.exports = function sanitize( raw, clean ){
...
-      raw['boundary.circle.lat'] = clean['point.lat'];
-    }
-    if( !clean.hasOwnProperty('boundary.circle.lon') && clean.hasOwnProperty('point.lon') ){
-      raw['boundary.circle.lon'] = clean['point.lon'];
+    // overwrite boundary.circle.lat/lon with point.lat/lon
+    raw['boundary.circle.lat'] = clean['point.lat'];
+    raw['boundary.circle.lon'] = clean['point.lon'];
+
+    // if no radius was passed, set the default
+    if (!check.assigned(raw['boundary.circle.radius'])) {
```
Comment (dianashk) [inline sanitiser/_geo_reverse.js]: use `check.undefined` instead to avoid using the `!`

**Comment by dianashk** [comment]
I merged master into this branch to fix travis failures. Code looks good. Just the minor comments from me. Other than that, I'm happy for it to merge:

:rocket:

**Comment by stephenkhess** [comment]
updated per Diana's suggestions

**Comment by dianashk** [comment]
:ship:

**Pull request merged**